amethyst_assets/src/asset.rs

```diff
@@ -123,6 +133,26 @@ impl<D: 'static> Format<D> for Box<dyn Format<D>> {
     }
 }
 
+impl<D: 'static> Format<D> for Box<dyn SerializableFormat<D>> {
```

Comment:
Do we need such an impl for `dyn SerializableFormat<D>`, too?

Reply:
Yeah... I think, we still need to use `Box<dyn SerializableFormat<D>>` as `Format`. Otherwise, without this implementation, we get
```
error[E0277]: the trait bound `std::boxed::Box<(dyn asset::SerializableFormat<D> + 'static)>: asset::Format<D>` is not satisfied
   --> amethyst_assets\src\asset.rs:136:41
    |
136 | impl<D: FormatRegisteredData + 'static> SerializableFormat<D> for Box<dyn SerializableFormat<D>> {}
    |                                         ^^^^^^^^^^^^^^^^^^^^^ the trait `asset::Format<D>` is not implemented for `std::boxed::Box<(dyn asset::SerializableFormat<D> + 'static)>`
    |
    = help: the following implementations were found:
              <std::boxed::Box<(dyn asset::Format<D> + 'static)> as asset::Format<D>>
```
I'm not sure there's any other way to do it. Please correct me if I'm wrong